src/web3-contracts.js

```diff
@@ -305,16 +303,19 @@ export function useConvertTokenToAnj(selectedToken) {
       // If the user has selected ANT, we can directly
       // approve and call the wrapper using ANT's approveAndCall
       if (selectedToken === 'ANT') {
-        const encodedActivation = ethers.utils.hexlify(bigNum(1)).split('0x')[1]
-        const encodedMinTokens = ethers.utils
-          .hexlify(underestimatedAnj)
-          .split('0x')[1]
-        const encodedMinEth = ethers.utils.hexlify(estimatedEth).split('0x')[1]
-        const encodedDeadline = ethers.utils
-          .hexlify(twoHourExpiry)
-          .split('0x')[1]
+        const encodedActivation = '1'.padStart(64, '0')
+        const encodedMinTokens = bigNum(underestimatedAnj)
+          .toHexString()
```

Comment:
Let's leave a few comments here giving a short explanation.

We're padding to 64 to create a 32-byte word, and we use `.slice(2)` because `.toHexString()` returns `0x...`.